src/app/onprem/license-nag-message-management/nag-message.html

```diff
@@ -11,10 +11,9 @@
 
 -->
 <div class="license-message">
-  You Codenvy does not have a valid subscription. Codenvy is free for 10 users and a single server.&nbsp;
+  Your Codenvy does not have a valid subscription. Codenvy is free for 5 users and a single server.&nbsp;
```

Comment:
this number of users is better to move to constant and to be changed at one place